CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## DEV
+
+* Add `models.Zone.get_ssl_verification_info` and corresponding service
+  support.
```

Comment:
I would add separate log entries for service method and model's method.  But that's not critical.